.github/workflows/test-dbt-installation-docker.yml

```diff
@@ -110,13 +114,22 @@ jobs:
         run: |
           docker image ls --all
 
+      - name: "Verify dbt-core Version"
+        if: ${{ inputs.package_name == 'dbt-core' }}
+        uses: addnab/docker-run-action@v3
+        with:
+          registry: ${{ env.GITHUB_PACKAGES_LINK }}
+          image: ${{ steps.image-info.outputs.ref }}
+          run: dbt --version
+
       - name: "Verify ${{ inputs.package_name }} Version"
+        if: ${{ inputs.package_name != 'dbt-core' }}
         uses: addnab/docker-run-action@v3
         with:
           registry: ${{ env.GITHUB_PACKAGES_LINK }}
           image: ${{ steps.image-info.outputs.ref }}
           run: |
-            PACKAGE = $(echo ${{ inputs.package_name }} | cut -c 5-)
+            PACKAGE=$(echo ${{ inputs.package_name }} | cut -c 5-)
```

Comment:
Any chance this could fail without double quotes surrounding the subshell?

Reply:
Oh, not sure, but the run off of this branch was successful, and `inputs.package_name` will always look roughly the same.